server/views/pages/reviewPlan/reviewNote/index.njk

```diff
@@ -23,12 +37,11 @@
             value: form.notes,
             label: {
               text: "Add a note to this review",
-              classes: "govuk-label--l",
+              classes: "govuk-visually-hidden",
               isPageHeading: true
```

Comment:
I think you'll want to remove the `isHeading` attribute. I think with it set to `true` then you'll end up with 2 `<h1>` elements on the page (the first one will be the new one you've added above the govk warning text; and the 2nd one will be this one with the visually hidden styling)
Without the `isHeading` attribute I _think_ it will render it as a `<legend>` element, which is probably more semantically correct than 2 `<h1>`s on the page